### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by TreyE**
Adds github actions with our standard RSpec and RuboCop settings.

Includes corrections to pass rubocop.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,22 +1,128 @@
...
+# This yaml describes our current checks.
 AllCops:
+  TargetRubyVersion: 2.6
+  SuggestExtensions: false
   NewCops: enable
+  Exclude:
+    - "bin/**/*"
+    - "vendor/**/*"
+
+Layout/CommentIndentation:
```
Comment (ipublic) [inline .rubocop.yml:10]: My understanding is with this initialization Rubocop uses the idiomatic Ruby guide by default.  Therefore we only need to indicate the Enabled: false exclusions.  

How are we implementing plan to only check for complex methods?

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+
 source "https://rubygems.org"
 
 # Specify your gem's dependencies in aca_entities.gemspec
 gemspec
 
 gem "rake", "~> 12.0"
 gem "rspec", "~> 3.0"
+
+gem "rubocop"
```
Comment (ipublic) [inline Gemfile:11]: Wrap these in 

group :development, :test do
  gem "rubocop"
  gem "yarn"
end

File: lib/aca_entities/error.rb [hunk 3]
```diff
@@ -7,7 +7,7 @@ module Error
     module ErrorInitalizer
       attr_reader :original
 
-      def initialize(msg, original = $!)
+      def initialize(msg, original = $ERROR_INFO)
```
Comment (ipublic) [inline lib/aca_entities/error.rb:10]: pulled this code from Avdi guidance.  it appears $ERROR_INFO is functionally equivalent to $! but in context see it requires:

required 'English'

File: lib/aca_entities/iap/iap.rb [hunk 4]
```diff
@@ -8,7 +9,6 @@
 require_relative 'benefit'
 require_relative 'deduction'
 require_relative 'eligibility_determination'
-require_relative 'employer'
```
Comment (ipublic) [inline lib/aca_entities/iap/iap.rb:11]: Why was this reference removed?

**Review by ipublic — CHANGES_REQUESTED** [review]
Open to discuss comments where you disagree

File: lib/aca_entities/questionnaires/types.rb [hunk 5]
```diff
@@ -10,8 +10,8 @@ module Types
       send(:include, Dry.Types())
       send(:include, Dry::Logic)
 
-      VersionKind = Types::Coercible::String.default('0.1.0'.freeze).enum('0.1.0')
-      DataTypeKind = Types::Coercible::String.enum()
+      VersionKind = Types::Coercible::String.default('0.1.0').enum('0.1.0')
```
[the remark this answers is not in the document]
Comment (TreyE) [reply lib/aca_entities/questionnaires/types.rb:13]: Not needed as "# frozen_string_literal: true" is the default and is in the header of the file.

**Pull request closed without merging**